linera-service/src/linera.rs

```diff
@@ -1345,12 +1365,11 @@ impl Runnable for Job {
                 let balance = chain_client
                     .local_balance()
                     .await
-                    .expect("Use sync_balance instead");
+                    .context("Use sync_balance instead")?;
                 let time_total = time_start.elapsed().as_micros();
```

Comment:
Note that the time elapsed may or may not include saving the wallet now.

Reply:
Why? It looks like it never includes it? And I didn't change that behavior; I just replaced the panic with an error.

Reply:
Ah, you mean in the other commands. Yes, maybe I should include it here, too.